roles/influxdb/tasks/bucket.yml

```
@@ -0,0 +1,46 @@
+---
+- name: Install Influx DB 2 CLI
+  package: 
```

```suggestion
  package:
```
Linter doesn't like trailing spaces. :-)